.github/workflows/clp-rust-checks.yaml

```
@@ -31,6 +31,10 @@ jobs:
         shell: "bash"
         run: "npm install -g @go-task/cli@3.44.0"
 
+      - name: "Deps"
```

would
```suggestion
      - name: "Verify Locks"
```
be more clear? Otherwise it might look like it's a deps installation step